src/prisma/index.ts

```diff
@@ -13,3 +13,27 @@ async function lookupUser(username: string) {
   console.log(user);
   return user;
 }
+
+async function lookupId(userid: number) {
```

Comment:
Perhaps we either move these to a `src/users/index.ts` or we make these a bit more descriptive?
ie. `findUserById(id: number)`?

Reply:
I think the function names make sense like lookupUser would be looking up a username. Maybe for courses and schedule I could make it findCourses or findSchedule.

Reply:
I prefer findUserByID. You can also refactor lookupUser to findUserByUsername(and move it too).

You could consider making another file within the prisma folder for utility functions `src/prisma/utils.ts` or even split up all your functions into various subfiles for  similar functions like `prisma/users.ts`, `prisma/schedules.ts` and so on.